test/parallel/test-http2-respond-file-304.js

```diff
@@ -12,7 +13,7 @@ const {
   HTTP2_HEADER_STATUS
 } = http2.constants;
 
-const fname = path.resolve(common.fixturesDir, 'elipses.txt');
+const fname = path.resolve(fixtures.fixturesDir, 'elipses.txt');
```

Comment:
You should be able to use `fixtures.path('elipses.txt')` rather than the `path.resolve`, which would be even nicer. If you could push a new commit to your branch to add this change that'd be ideal.

Not mandatory though.

See the docs for more info: https://github.com/nodejs/node/blob/master/test/common/README.md#fixturespathargs

Reply:
Oh, cool. Thanks for the heads up. I'll try and get a new commit up as soon as I get the chance.